sdk/appconfiguration/feature-management/README.md

```diff
@@ -0,0 +1,236 @@
+# Feature Management client library for Python
+
+Feature Management is a library for enabling/disabling features at runtime. Developers can use feature flags in simple use cases like conditional statement to more advanced scenarios like conditionally adding routes.
+
+## Getting started
+
+### Install the package
+
+Install the Azure Feature Management client library for Python with [pip][pip]:
+
+```bash
+pip install feature-management
+```
+
+### Prerequisites
+
+* Python 3.7 or later is required to use this package.
+
+## Examples
+
+You can use feature flags from the Azure App Configuration service, a json file, or a dictionary.
+
+#### Use feature flags from Azure App Configuration
+
+```python
+from azure.appconfiguration.provider import load
+from azure.identity import DefaultAzureCredential
+import os
+
+endpoint = os.environ.get("APPCONFIGURATION_ENDPOINT_STRING")
+
+selects = {SettingSelector(key_filter=".appconfig.featureflag*")}
+
+config = load(endpoint=endpoint, credential=DefaultAzureCredential(), selects=selects)
+
+feature_manager = FeatureManager(config)
+
+# Is always true
+print("Alpha is ", feature_manager.is_enabled("Alpha"))
+```
+
+NOTE: If no setting selector is set then feature flags with no label are loaded.
+
+### Use feature flags from a json file
+
+```python
+from feature.management import FeatureManager
+import json
+import os
+import sys
+
+script_directory = os.path.dirname(os.path.abspath(sys.argv[0]))
+
+f = open(script_directory + "/my_json_file.json", "r")
+
+feature_flags = json.load(f)
+
+feature_manager = FeatureManager(feature_flags)
+
+# Is always true
+print("Alpha is ", feature_manager.is_enabled("Alpha"))
+```
+
+### Use feature flags from a dictionary
+
+```python
+from feature.management import FeatureManager
+
+feature_flags = {
+    "FeatureFlags": [
+        {"id": "Alpha", "enabled": "true"},
+    ]
+}
+
+feature_manager = FeatureManager(feature_flags)
+
+# Is always true
+print("Alpha is ", feature_manager.is_enabled("Alpha"))
+```
+
+## Key concepts
+
+### FeatureManager
+
+The `FeatureManager` is the main entry point for using feature flags. It is initialized with a dictionary of feature flags, and optional feature filters. The `FeatureManager` can then be used to check if a feature is enabled or disabled.
+
+### Feature Flags
+
+Feature Flags are objects that define how Feature Management enables/disables. It contains an `id` and `enabled` property. The `id` is a string that uniquely identifies the feature flag. The `enabled` property is a boolean that indicates if the feature flag is enabled or disabled. The `conditions` property is a list of `FeatureFilter` objects that are used to determine if the feature flag is enabled or disabled.
```

Comment:
Feature Flags are objects that define how Feature Management enables/disables "a feature"?